change/@fluentui-react-charting-6741ba08-9ad2-46d0-90a6-298da205529f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Add the details of this change in PR description

Reply:
Done